framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java

```diff
@@ -348,16 +349,38 @@ private static int getMissingCount(Map<ColumnList, Integer> map) {
   }
 
   private static int compareTo(ColumnList list1, ColumnList list2,
-      Map<Integer, String> columnIndexAndOrder, int start) {
```

Comment:
There are comments for verifyResultSetOrders that provide some of the rules.  Is this sufficient?  I can add some example queries.